lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -128,6 +128,51 @@ class EngineAccessibilityFeatures implements ui.AccessibilityFeatures {
   }
 }
 
+class EngineAccessibilityFeaturesBuilder {
+  EngineAccessibilityFeaturesBuilder(this._index);
```

Comment:
Using this class the implementation of `EngineAccessibilityFeatures.copyWith` can be simplified.